parla/cuda.py

```diff
@@ -167,6 +167,16 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.stream.synchronize()
         return self.gpu.cupy_device.__exit__(exc_type, exc_val, exc_tb)
 
+    def initialize_thread(self) -> None:
+        for gpu in self.gpus:
+            # TODO(ian): Trigger cblas/etc. initialization for this GPU in this thread.
```

Comment:
Seems like this isn't a TODO any more. Delete it. Same below.

Reply:
Will do.